### PR TITLE
Disable problematic SELinux code causing runc issues

### PR DESCRIPTION
**Pull request by mheon**
Temporarily disable some SELinux labelling code that was causing major runc issues

### CODE REVIEW

**Comment by baude** [comment]
LGTM, though I would like to see the issue number recorded in the code

**Comment by TomSweeneyRedHat** [comment]
This change went  in with #1524 which add mount to run.  Should we back that out, or perhaps back out the cli flag so it can't be called?

**Comment by mheon** [comment]
This change should be independent of the remainder of the mount flag code, which is why I'm only removing this part

File: libpod/container_internal_linux.go [hunk 1]
```diff
@@ -283,13 +283,15 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
...
 	g.ClearMounts()
 	for _, m := range mounts {
-		switch m.Type {
-		case "tmpfs", "devpts":
-			o := label.FormatMountLabel("", c.config.MountLabel)
-			if o != "" {
-				m.Options = append(m.Options, o)
-			}
-		}
+		// TODO fix and reenable this code
```
Comment (rhatdan) [inline libpod/container_internal_linux.go]: Just delete this section.  Still have an issue, but this is not fixing it.

**Comment by mheon** [comment]
Done

**Comment by mheon** [comment]
@rh-atomic-bot r=baude

**Pull request closed without merging**